ccd-definition/CaseEvent.json

```diff
@@ -295,9 +295,20 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "Submitted",
     "PostConditionState": "Submitted",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "uploadDocumentsAfterSubmission",
+    "Name": "Documents",
+    "Description": "Upload a court bundle",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "Submitted",
+    "PostConditionState": "Submitted",
     "SecurityClassification": "Public",
     "ShowSummary": "N",
     "ShowEventNotes": "N",
-    "EndButtonLabel": "Save and continue"    
+    "EndButtonLabel": "Save and continue"
```

Comment:
That should be part of `uploadStandardDirections` event otherwise E2E tests will fail.

Also double check with business what button wording is needed for `uploadDocumentsAfterSubmission`.

Reply:
Changes made and pushed and e2e tests updated